lib/repl.js

```diff
@@ -698,6 +699,10 @@ function filteredOwnPropertyNames(obj) {
   return Object.getOwnPropertyNames(obj).filter(intFilter);
 }
 
+REPLServer.prototype.complete = function() {
```

Comment:
The way I read the code, it seems this function was added to `REPLServer.prototype` in order to allow for simpler testing. The result is a a greater API surface area. Is there a need for this function to be available at the user level? If not, perhaps is there a way to accomplish the testing requirements without adding to the `prototype`.